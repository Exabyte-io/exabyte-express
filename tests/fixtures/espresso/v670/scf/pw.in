version https://git-lfs.github.com/spec/v1
oid sha256:b1e1e8804a988764ec99473961ded3abd3c791236d2803f60ba420649fdbb312
size 889
