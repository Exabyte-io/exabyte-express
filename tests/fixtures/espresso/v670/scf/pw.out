version https://git-lfs.github.com/spec/v1
oid sha256:729ac7b43e730e935e99146902719895d16c95a56e2a31b8d95c1fc8d4447fcf
size 9874
